src/frontend/components/UI/SelectField/index.tsx

```diff
@@ -35,13 +36,24 @@ const SelectField = ({
       })}
     >
       {label && <label htmlFor={htmlId}>{label}</label>}
-      <select id={htmlId} value={value} onChange={onChange} disabled={disabled}>
-        {prompt && <option value="">{prompt}</option>}
+      <Select
+        id={htmlId}
+        value={value}
+        onChange={onChange}
+        disabled={disabled}
+        sx={{
+          '& .MuiSelect-icon': { display: 'none' },
+          '& .MuiOutlinedInput-notchedOutline legend': { width: 0 },
+          '& .MuiSelect-select': {
+            textAlign: isRTL ? 'right' : 'left',
+            paddingLeft: 0
```

Comment:
can we use `paddingInlineStart: 0` here instead? it's better to support RTL languages since `inlineStart` is equivalent to `left` in LTR and `right` in RTL

Reply:
I've ended up just using MUI's padding instead of ours, which also resolves an issue with excessive padding on RTL text